djmoney/models/fields.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, verbose_name=None, name=None,
                  currency_field_name=None, **kwargs):
         nullable = kwargs.get('null', False)
         default = self.setup_default(default, default_currency, nullable)
-        if not default_currency:
+        if not default_currency and default is not None:
```

Comment:
What concerns me with this is that we allow assigning just number types to a Money field, and we might then have a database value that says `money=1.234` and `currency=null` ?